pygyro/splines/spline_eval_funcs.py

```diff
@@ -1,9 +1,4 @@
-from pyccel.decorators  import types, pure, stack_array
```

Comment:
Why is `@pure` and worse `@stack_array` removed?!

Reply:
They will be readded.